docs/compose/requirements.rst

```diff
@@ -23,16 +23,16 @@ the latest Linux kernel. The minimal required memory and swap are:
 Pick a distribution
 -------------------
 
-The mail server runs as a set of Docker containers. It is thus almost agnostic
-of the underlying operating system as long as a fairly recent Linux kernel is
-running and the Docker API (>= 1.11) is available.
+The mail server runs as a set of Docker containers, so is it almost operating 
```

Comment:
Shouldn't this be "so it is"?

Reply:
Good catch. It should indeed be `... , so it is almost ...`.
That is why we require 2 reviewers for a PR :wink:

Reply:
Fixed in 89cc6e0716ed16f28a198c986ca7f1ffab94bff7